CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 This is a changelog for Piwik platform developers. All changes for our HTTP API's, Plugins, Themes, etc will be listed here.
 
+## Piwik 3.0.0
+
+### Breaking Changes
```

Comment:
We will need to optimize the changelog at a later point once we have the final API's and some guides on developer.piwik.org . I expect some of those API's to change (once we work a bit more with it etc) before releasing Piwik 3.0